docs/layouts/models/single.html

```
@@ -48,109 +61,113 @@ <h3>
+            {{ range readDir $catalogTypePath }}
+            {{ if and (not .IsDir) (strings.HasSuffix .Name ".md") }}
+            {{ $filePath := printf "%s/%s" $catalogTypePath .Name }}
+            {{ $raw := readFile $filePath }}
+            {{ $parts := split $raw "---" }}
+            {{ if ge (len $parts) 3 }}
+            {{ $meta := transform.Unmarshal (index $parts 1) }}
+            {{ $compatibility := index $meta "compatibility" }}
+            {{ if and $compatibility (in $compatibility $tech) }}
             <li class="design-link">
-                <a href="https://meshery.io/catalog/{{ $catalogType }}/{{ replace (index $meta "name") "." "-" | urlize }}-{{ index $meta "patternId" }}" target="_blank">
+                <a href="https://meshery.io/catalog/{{ $catalogType }}/{{ replace (index $meta " name") "." "-" | urlize
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)